### ISSUE TITLE
sniping sources

### ISSUE THREAD

**Issue by madzul** [PokemonGoF/PokemonGo-Bot#6255]
Any usable sniping sources exist?

**Comment by MerlionRock** [comment]
not really.... most are now in the form of discord. You can try to use https://github.com/pogarek/PSSnipeFeeder

Not too sure if it's still working...

what it does is to crawl through discord channel and get the coordinates from discord and create the json file needed for sniping.

For sniping to work, it has to be within close proximity. The bot will automatically calculate time needed to wait before sniping. Suggestion is to stay within a city. Alternatively, you can use map like sgpokemap, make your bot roam in SG, and snipe using telegram

**Comment by madzul** [comment]
thankyou
how to snipe using telegram?
